pkg/core/native/std.go

```diff
@@ -204,7 +205,7 @@ func (s *Std) jsonDeserialize(_ *interop.Context, args []stackitem.Item) stackit
 		panic(err)
 	}
 
-	item, err := stackitem.FromJSON(data)
+	item, err := stackitem.FromJSON(data, vm.MaxStackSize)
```

Comment:
`stackitem.MaxDeserialized`? It's the same and is used already for binary deserialization.